documentation/ingestion_server/guides/running.md

```diff
@@ -0,0 +1,16 @@
+# Running on the Host
+
+1. Create environment variables from the template file.
+```
+just env
+```
+
+2. Install Python dependencies
+```
+just install
+```
+
+3. Start the Gunicorn server.
+```
+pipenv run gunicorn
+```
```

Comment:
Similar to the "deploy" page, I think we can remove this. We don't support or use this workflow at all (we always run the ingestion server in docker) and by documenting it we're implying that we do support it. We don't have time to maintain this specific workflow, so it's best to just ignore it and not document it.